war/src/main/webapp/images/build-status/weather-sprite.svg

```diff
@@ -0,0 +1,41 @@
+<svg width="0" height="0" class="hidden">
```

Comment:
We should acknowledge the source https://github.com/jenkins-contrib-themes/jenkins-core-theme/tree/master/images here; it's MIT licensed according to a metadata file. Not a lawyer but IIUC, while we can freely use it, we need to acknowledge where it's from and the license we're using it under.

This also needs adding to https://github.com/jenkinsci/jenkins/blob/b350b10da7800f6ddf107cf3aa9cd5d9c058ec43/core/src/main/resources/hudson/AboutJenkins/index.jelly#L52